src/chrome/content/rules/aliexpress.com-mixedcontent.xml

```diff
@@ -2,18 +2,13 @@
 	For rules not causing MCB, see AliExpress.com.xml.
 -->
 <ruleset name="AliExpress.com (MCB)" platform="mixedcontent">
+	<target host="aliexpress.com" />
```

Comment:
Are you still get MCB now?

Reply:
@ivysrono https://aliexpress.com/ is fine for me (from US IP).

Reply:
https://www.aliexpress.com is fine too?

Reply:
@ivysrono Yes.

Reply:
@pipboy96 oh, I find @injust has add test in ruleset: http://www.aliexpress.com/buyerprotection/overview.html
IMO, this PR could be merged now.